src/main/java/oss/fosslight/util/FileUtil.java

```diff
@@ -189,24 +189,25 @@ public static String makeValidFileName(String fileName, String replaceStr) {
 
 	public static void zip(String inputFolder, String filePath, String zipName, String zipRootEntryName) throws Exception {
 		// 압축파일을 저장할 파일을 선언한다.
-		FileOutputStream fileOutputStream = null;
 		File file = new File(filePath + File.separator + zipName);
-		fileOutputStream = new FileOutputStream(file);
-		// ZipOutputStream 선언
-		ZipOutputStream zipOutputStream = new ZipOutputStream(fileOutputStream);
-		// 압축할 대상이 있는 폴더를 파일로 선언한다.
-		File inputFile = new File(inputFolder);
-		
-		// 압축을 할 대상이 file이면 zipFile 메소드를,
-		// 폴더이면 zipFolder 메소드를 호출한다.
-		if (inputFile.isFile()) {
-			zipFile(zipOutputStream, inputFile, "");
-		} else if (inputFile.isDirectory()) {
-			zipFolder(zipOutputStream, inputFile, "", zipRootEntryName);
+		try(			
+			FileOutputStream fileOutputStream = new FileOutputStream(file);
+			// ZipOutputStream 선언
+			ZipOutputStream zipOutputStream = new ZipOutputStream(fileOutputStream);
+		) {
+			// 압축할 대상이 있는 폴더를 파일로 선언한다.
+			File inputFile = new File(inputFolder);
+			
+			// 압축을 할 대상이 file이면 zipFile 메소드를,
+			// 폴더이면 zipFolder 메소드를 호출한다.
+			if (inputFile.isFile()) {
+				zipFile(zipOutputStream, inputFile, "");
+			} else if (inputFile.isDirectory()) {
+				zipFolder(zipOutputStream, inputFile, "", zipRootEntryName);
+			}
+		} catch (Exception e) {
+			throw e;

```

Comment:
There is a part that is not wrapped with try catch in calling this function. (Ex- ./main/java/oss/fosslight/service/impl/VerificationServiceImpl.java)
Please check it out.

Reply:
I don't know what it is best practice but I see a hierarchy that 

getNoticeHtml try_catch(VerificationController) -> updateStatusWithConfirm throws(VerificationServiceImpl) -> zip throws

So In the controller, The error will be handled.

Reply:
Great.